yorkie/config.go

```diff
@@ -18,19 +18,50 @@ package yorkie
 
 import (
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"os"
 
 	"github.com/yorkie-team/yorkie/pkg/log"
 	"github.com/yorkie-team/yorkie/yorkie/backend/mongo"
 )
 
+const (
+	DefaultMongoDBURI     = "mongodb://localhost:27017"
+	DefaultYorkieDatabase = "yorkie-meta"
+)
+
 type Config struct {
 	RPCPort int
 	Mongo   *mongo.Config
 }
 
-func NewConfig(path string) (*Config, error) {
+func (c *Config) RPCAddr() string {
+	return fmt.Sprintf("localhost:%d", c.RPCPort)
+}
+
+func newConfig(port int, dbname string) *Config {
+	return &Config{
+		RPCPort: port,
+		Mongo: &mongo.Config{
+			ConnectionURI:        DefaultMongoDBURI,
+			ConnectionTimeoutSec: 5,
+			PingTimeoutSec:       5,
+			YorkieDatabase:       dbname,
+		},
+	}
+}
+
+func NewConfig() *Config {
+	const port = 9090
```

Comment:
What about extracting `port = 9090` as a constant `DefaultPort` on the code side below?
https://github.com/yorkie-team/yorkie/pull/15/files#diff-adef28dace684692d8d4863aca3705fcR31